### PR TITLE
Added License Header in all files

### PR DESCRIPTION
**Pull request by dhruvpeter**
#7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+###### (C) Copyright ${year} core.ai (https://core.ai/)
```
Comment (abose) [inline README.md:1]: GPL clause is only necessary for code files, ie. *.py files. It can be omitted for all other files.

**Review by abose — CHANGES_REQUESTED** [review]

**Comment by abose** [comment]
@dhruvpeter Thanks for submitting the changes.

File: README.md (hunk 1 above)
Comment (dhruvpeter) [reply README.md:1]: What about .yaml files, .toml files, and .cfg files?

Comment (abose) [reply README.md:1]: We can omit those too. It might be problematic if someone uses custom parsers for config files that do not support comments.

**Comment by dhruvpeter** [comment]
Requested changes made

File: setup.py [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+# Contributors: Author 1, author 2...
```
Comment (abose) [inline setup.py:16]: you could add your name here or could specify `contributors: your name and others[see commit log]`

File: setup.py [hunk 3]
```diff
@@ -1,3 +1,20 @@
+# (C) Copyright ${year} core.ai (https://core.ai/)
```
Comment (abose) [inline setup.py:1]: There are 4 duplicate commits in the PR.  It is a good practice to squash redundant commits.

https://blog.carbonfive.com/always-squash-and-rebase-your-git-commits/

**Review by abose — CHANGES_REQUESTED** [review]

File: setup.py (hunk 2 above)
Comment (dhruvpeter) [reply setup.py]: Done

File: setup.py (hunk 3 above)
Comment (dhruvpeter) [reply setup.py:1]: Done

**Comment by dhruvpeter** [comment]
Requested changes made

**Comment by abose** [comment]
LGTM Merged.

**Pull request merged**